docs/docs/casper/sdk.md

```diff
@@ -233,12 +279,305 @@ The deploy function serves as a one-stop solution for deploying ERC20 contracts.
 By providing the contract hash returned during deployment, the load function dynamically adds all necessary methods to the specified contract. This dynamic loading capability ensures that the contract is equipped with the essential functionalities defined by the ERC20 standard.
 :::
 
-Load the Invariant contract by providing the Casper Client (`client`) and Casper Serice (`service`), and indicating the Invariant contract hash (`INVARIANT_CONTRACT_HASH`). Similarly, initialize the ERC20 token contract using the same approach.
+Load the Invariant contract by providing the Casper Client (`client`), and indicating the Invariant contract hash (`INVARIANT_CONTRACT_HASH`). Similarly, initialize the ERC20 token contract using the same approach.
 
 ```typescript
 // load invariant contract
-const invariant = await Invariant.load(client, service, INVARIANT_CONTRACT_HASH)
+const invariant = await Invariant.load(client, INVARIANT_CONTRACT_HASH, Network.Local)
 
 // load token contract
-const erc20 = await Erc20.load(client, service, ERC20_CONTRACT_HASH)
+const erc20 = await Erc20.load(client, Network.Local, TOKEN_0_CONTRACT_HASH)
+```
+
+### Create pool
+
+:::info Big numbers
+You can create custom decimals using the `toDecimal` syntax, where the first argument represents the numerical value (A), and the second argument indicates the power of 10 (B) in the formula `A * 10^(-B)`. For example, `toDecimal(3n, 2n)` will result in a decimal equal to 0.03. For further details on supported types, please check the documentation [here](types.md).
+:::
+
+:::note Why "n" is at the end of every number
+Notice how we use "n" at the end of every number. "n" indicates that specified value is a BigInt, number with higher precision.
+:::
+
+:::warning Token sorting
+Tokens are sorted alphabetically when pool key is created, so make sure that you swap tokens in correct direction. Read more about pool keys [here](storage#poolkey).
+:::
+
+To create a new pool, a fee tier and pool key need to be prepared. The fee tier represents the desired fee for the pool, and the price needs to be converted to sqrt price because the entry points of the protocol accept it in this format. The pool key is constructed using the addresses of two tokens and the specified fee tier. Finally, the `createPool` function is called with the user's account, the pool key, and the initial square root price, resulting in the creation of a new pool. The transaction hash of the pool creation is then logged to the console.
+
+```typescript
+// set fee tier, make sture that fee tier with specified parameters exists
+const feeTier = await newFeeTier(toDecimal(Decimal.SqrtPrice, 1n, 2n), 1n) // fee: 0.01 = 1%, tick spacing: 1
+
+// if the fee tier does not exist, you have to add it
+const isAdded = await invariant.feeTierExist(feeTier)
+if (!isAdded) {
+  await invariant.addFeeTier(account, feeTier)
+}
+
+// set initial price of the pool, we set it to 1.00, remember its square root of price
```

Comment:
I would adjust the comment and stick only to mentioning "sqrt price" and get rid of "price"
`set initial sqrt price of the pool to 1.00`